src/main/scala/org/opennetworkinsight/FlowColumnIndex.scala

```diff
@@ -1,20 +1,22 @@
 package org.opennetworkinsight
 
 object FlowColumnIndex extends Enumeration {
-    val HOUR = 0
-    val MINUTE = 1
-    val SECOND = 2
-    val SOURCEIP = 3
-    val DESTIP = 4
-    val SOURCEPORT =5
-    val DESTPORT = 6
-    val IPKT = 7
-    val IBYT = 8
-    val NUMTIME = 9
-    val IBYTBIN = 10
-    val IPKTYBIN = 11
-    val TIMEBIN = 12
-    val SOURCEWORD = 13
-    val DESTWORD = 14
+    val HOUR = 4
```

Comment:
After changing indexes for ML to read avro parquet data in the first attempt SOURCEWORD and DESTWORD indexes were misplaced.
 
I'm also including back all the columns in flow table as they are needed by Flow OA and Feedback.

Assigned the correct indexes and tested with a run of a complete day of data.

words.data, doc.dat and model.dat are looking like they are supposed to look.